src/IntInRangeParser.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class IntInRangeParser extends AbstractInRangeParser
+{
+    protected function describe()
+    {
+        return "an integer value between $this->minValue and $this->maxValue";
```

Comment:
"an integer value between ..." => "an integer between ...", this would also sound more human to me (e.g. when talking about integers, I never say "the integer value", usually just "the integer")